test/statsd/default_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build !windows
+
+package statsd
+
+import (
+	"github.com/aws/amazon-cloudwatch-agent-test/test/metric"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/test_runner"
+	"strings"
+	"time"
+)
+
+const testRetryCount = 3
+
+type StatsDRunner struct {
+	test_runner.BaseTestRunner
+	testName     string
+	dimensionKey string
+}
+
+func (t *StatsDRunner) Validate() status.TestGroupResult {
+	metricsToFetch := t.GetMeasuredMetrics()
+	testResults := make([]status.TestResult, len(metricsToFetch))
+
+	// ECS taskdef with portMappings has some delays before getting metrics from statsd container
+	if strings.Contains(t.testName, "ECS") {
+		time.Sleep(30 * time.Second)
+	}
```

Comment:
I would prefer if there was a way to parameterize this within the test outside of just inspecting the name - worried about whether this needs to be expanded. But I'm not gonna block on it this time around.